### DISCUSSION TITLE
Supervisor su nuova installazione

### DISCUSSION THREAD

**Discussion by matteo-vannucchi** [indomus/forum#330] [HUB: Home Assistant]
Salve, mi chiamo Matteo e leggo i  posti di inDomus da  qualche tempo ed ho acquistato un broadlink rm4 minipiù alcune prese smart Meross per iniziare  a fare alcune prove con la domotica.

Mi sono deciso ad installare HA core su docker sul mio portatile con Ubuntu, per prendere  confidenza con il software  e le relative configurazioni prima di acquistare un raspberry pi e passare a scenari di automazione più complessi.

Ho seguito le guide pubblicate sul sito, ma nella  mia installazione non riesco a trovare il supervisor e nemmeno istruzioni su come installarlo. Mi potete aiutare?
Queste  sono le informazioni sul mio sistema:
## System Health

version | core-2021.4.3
-- | --
installation_type | Home Assistant Container
dev | false
hassio | false
docker | true
virtualenv | false
python_version | 3.8.7
os_name | Linux
os_version | 5.4.0-73-generic
arch | x86_64
timezone | Europe/Rome

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
Github API Calls Remaining | 4962
Installed Version | 1.11.3
Stage | running
Available Repositories | 822
Installed Repositories | 2

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Lovelace</summary>

dashboards | 1
-- | --
resources | 0
views | 2
mode | storage

</details>

**Comment by KC-inDomus** [comment]
ciao Matteo e benvenuto. è normale che tu non lo trovi: 

https://indomus.it/focus/home-assistant-hassio-o-home-assistant-core-quale-installazione-fa-al-caso-mio/

**Reply by matteo-vannucchi** [reply]
Quindi, se volessi installare l'add-on per Google Drive e i backup, HASSIO sarebbe l'unica soluzione? Ho provato a cercare  in giro, ma mi sembra che sia così

**Reply by KC-inDomus** [reply]
corretto, a meno di voler installare singolarmente quegli elementi via supervisor.
il discorso è: se sei in grado di installare core su docker su github, non hai bisogno di usare quegli addon. hai la libertà di realizzare progressivamente un'ottima e performante configurazione senza di essi, pur ottenendo i risultati che cerchi (backup eccetera)
